.editorconfig

```diff
@@ -283,6 +283,7 @@ dotnet_diagnostic.SA1649.severity = none # SA1649FileNameMustMatchTypeName - it
 dotnet_diagnostic.SA0001.severity = none # SA0001XmlCommentAnalysisDisabled
 dotnet_diagnostic.SA1101.severity = none
 dotnet_diagnostic.SA1124.severity = none
+dotnet_diagnostic.SA1129.severity = none
```

Comment:
Can we add a comment about what it is we're disabling?

Reply:
Added